src/index.ts

```diff
@@ -15,18 +17,33 @@ import {
   TotalValueLockResponse,
 } from './types/response'
 
+export {
+  Chain,
+  DefiPositionResponse,
+  DefiProfitsResponse,
+  PositionType,
+  Protocol,
+}
+
 export async function getPositions({
   userAddress,
   filterProtocolIds,
   filterChainIds,
+  blockNumbers,
 }: {
   userAddress: string
   filterProtocolIds?: Protocol[]
   filterChainIds?: Chain[]
+  blockNumbers?: Record<string, number | undefined>
 }): Promise<DefiPositionResponse[]> {
   const runner = async (adapter: IProtocolAdapter) => {
+    const chainId = adapter.getProtocolDetails().chainId
```

Comment:
```suggestion
    const chainId = adapter.chainId
```

`chainId` is a property of `IProtocolAdapter`.